src/org/labkey/test/components/ui/grids/DetailDataPanel.java

```diff
@@ -64,9 +71,53 @@ public DetailTableEdit clickEdit()
                 .withTitle("Editing " + title).waitFor();
     }
 
+    /**
+     * Get the table that has the details fields (i.e. column data) for this item. If this is an aliquot sample this
+     * table will have the fields from the parent sample.
+     *
+     * @return A {@link DetailTable}.
+     */
     public DetailTable getTable()
     {
-        return  elementCache().detailTable;
+        List<DetailTable> tables = elementCache().detailTables();
+
+        // As of release 21.05 the table with the column values for the sample is the last table in the panel.
+        return  tables.get(tables.size() - 1);
+    }
+
+    /**
+     * If this is an aliquot sample there will be multiple tables in the panel, the first table will contain the aliquot
+     * information (under the 'Aliquot Data' header). This will return that table.
+     *
+     * @return A {@link DetailTable}.
+     */
+    public DetailTable getAliquotTable()
+    {
+        List<DetailTable> tables = elementCache().detailTables();
+
+        if(tables.size() == 1)
+            throw new RuntimeException("There does not appear to be an 'Aliquot' table in this panel.");
```

Comment:
Consider throwing `NoSuchElementException` like you did with `DetailTable.getField`
```suggestion
            throw new NoSuchElementException("There does not appear to be an 'Aliquot' table in this panel.");
```

Reply:
Changed.